src/test/java/org/gridsuite/explore/server/ExploreTest.java

```diff
@@ -250,9 +266,12 @@ public MockResponse dispatch(RecordedRequest request) {
                     return new MockResponse().setResponseCode(200);
                 } else if (path.matches("/v1/identifier-contingency-lists/.*") && "PUT".equals(request.getMethod())) {
                     return new MockResponse().setResponseCode(200);
+                } else if (path.matches("/v1/parameters/" + PARAMETERS_UUID) && "POST".equals(request.getMethod())) {
+                    return new MockResponse().setBody(newElementUuidAsString).setResponseCode(200)
+                            .addHeader("Content-Type", "application/json; charset=utf-8");
                 } else if (path.matches("/v1/parameters.*")) {
                     return new MockResponse().setResponseCode(200);
-                } else if (path.matches("/v1/network-modifications/duplicate")) {
+                } else if (path.matches("/v1/network-modifications")) {
                     return new MockResponse().setBody(modificationIdsAsString).setResponseCode(200).addHeader("Content-Type", "application/json; charset=utf-8");
```

Comment:
How could we expect a duplication mock endpoint return the same UUID as the duplicated entity, this is madness ! :D

Reply:
yes, fixed